--- libffi.pc.in.orig	2017-04-07 20:50:37.884080471 +0900
+++ libffi.pc.in	2017-04-07 20:50:53.521861135 +0900
@@ -2,10 +2,10 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 toolexeclibdir=@toolexeclibdir@
-includedir=${libdir}/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
+includedir=@includedir@
 
 Name: @PACKAGE_NAME@
 Description: Library supporting Foreign Function Interfaces
 Version: @PACKAGE_VERSION@
 Libs: -L${toolexeclibdir} -lffi
-Cflags: -I${includedir}
+Cflags:

--- include/Makefile.in.orig	2017-04-07 20:49:34.298831098 +0900
+++ include/Makefile.in	2017-04-07 20:50:14.620318570 +0900
@@ -314,7 +314,7 @@
 AUTOMAKE_OPTIONS = foreign
 DISTCLEANFILES = ffitarget.h
 EXTRA_DIST = ffi.h.in ffi_common.h
-includesdir = $(libdir)/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
+includesdir = $(includedir)
 nodist_includes_HEADERS = ffi.h ffitarget.h
 all: all-am
 
